fe/fe-core/src/main/java/org/apache/doris/planner/DistributedPlanner.java

```diff
@@ -97,11 +97,18 @@ public DistributedPlanner(PlannerContext ctx) {
             isPartitioned = true;
         }
         long perNodeMemLimit = ctx_.getQueryOptions().mem_limit;
+        double autoBroadcastJoinThresholdPercentage = ctx_.getQueryOptions().getAutoBroadcastJoinThreshold();
```

Comment:
It seams that only hash join node uses the 'autoBroadcastJoinThreshold'.
So, It is better to hide the parameter acquisition inside the function 'createHashJoinFragment'

Reply:
yes, implement in here, only because the original code write in this way.

Reply:
make `autoBroadcastJoinThreshold` as DistributedPlanner's attribute to avoid double computing it